src/main/java/org/prebid/server/auction/mediatypeprocessor/MultiFormatMediaTypeProcessor.java

```diff
@@ -78,10 +80,11 @@ private MediaType preferredMediaType(BidRequest bidRequest,
                                          Account account,
                                          String originalBidderName,
                                          String resolvedBidderName) {
+
         return Optional.ofNullable(bidRequest.getExt())
                 .map(ExtRequest::getPrebid)
                 .map(ExtRequestPrebid::getBiddercontrols)
-                .map(bidders -> bidders.get(originalBidderName))
+                .map(bidders -> getBidder(originalBidderName, bidders))
```

Comment:
If getBidder returns null, it will cause an NPE on the next line. 
While project logic suggests this shouldn’t happen, it would be better for readability if you added a nonNull filter or some other safeguard.

Reply:
it won't since it's nullable optional that execute orElse in that case